doc/nrf/config_and_build/dfu/nrf70_fw_patch_update.rst

```diff
@@ -0,0 +1,188 @@
+For instructions on configuring basic partitions with the Partition Manager, refer to the:ref:`Partition Manager guide <partition_manager>`.
+
+This guide is predicated on the use of the Partition Manager and its associated configuration files.
+
+Defining nRF70 series MCUBoot partitions
+----------------------------------------
+
+The number of MCUBoot partitions required depends on the platform being used and its memory layout.
+In addition to a partition dedicated to the nRF70 Series firmware patch, you will need to create specific partitions based on the platform.
+For applications that are based on a single image, define a new partition with the number 1.
+For multi-image builds, define the subsequent partition to follow the last existing one.
+
+Below are examples of MCUBoot partition names for updating the nRF70 Series firmware patch, which vary depending on the platform and the number of cores used:
+
+* For the nRF5340 DK in a single-core variant (without the network core): ``mcuboot_primary_1`` and ``mcuboot_secondary_1``.
+* For the nRF5340 DK in a multi-core variant (with the network core): ``mcuboot_primary_2`` and ``mcuboot_secondary_2``.
+
+.. _nrf70_fw_patch_update_adding_partitions:
+
+Adding nRF70 series firmware patch partitions
+---------------------------------------------
+
+The examples below assume that there are two existing MCUBoot partitions (for the Application and Network Cores) and that the starting address of the free external memory space is ``0x12f000``.
+
+To add the required partitions for the nRF70 Series firmware patch update, complete the following steps:
+
+1. Create a partition for the MCUBoot header. This partition should start from the first available address in the external memory space and have a size equal to the value of the :kconfig:option:`CONFIG_IMG_BLOCK_BUF_SIZE` kconfig option.
+
+   For example:
+
+.. code-block:: console
+
+    nrf70_wifi_fw_mcuboot_pad:
+        address: 0x12f000
+        size: 0x200
+        device: MX25R64
+        region: external_flash
+
+#. Create the ``nrf70_wifi_fw partition``, which should start from the end address of the previously created MCUBoot header partition.
+   This partition must have a size of 128 kB (0x20000).
+
+   For example:
+
+.. code-block:: console
+
+    nrf70_wifi_fw:
+        address: 0x12f200
+        size: 0x20000
+        device: MX25R64
+        region: external_flash
+
+#. Create the ``mcuboot_primary_X`` partition, where ``X`` represents the appropriate partition number as described previously.
+   This partition should have the same starting address as the ``nrf70_wifi_fw_mcuboot_pad`` partition, and a size of 132 kB + 200 B aligned to the device's sector size.
+   It includes both the MCUBoot pad and the nRF70 Series firmware patch.
+
+   For example, the MX25R64 device has a sector size of 4 kB, so the following configuration can be used:
+
+.. code-block:: console
```

Comment:
Likewise for this code-block.

And I think I'll just stop adding this comment to each and every one of them. :)